### PR TITLE
Loki

### PR DESCRIPTION
**Pull request by pb82**
## What

Install and uninstall targets for Observatorium.

## Why

To be able to run our own instance of observatorium.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,46 @@ To specify which namespace strimzi & kafka are in, run the cmd with the followin
...
+
+```
+DEX_ROUTE=$(oc get routes dex -ndex -ojsonpath={.spec.host})
+```
+
+2) Request a token:
+
+```
+curl --request POST \
+              --url http://$(DEX_ROUTE)/dex/token \
```
Comment (david-martin) [inline README.md:131]:
```suggestion
              --url http://$DEX_ROUTE/dex/token \
```

File: README.md [hunk 2]
```diff
@@ -100,6 +100,46 @@ To specify which namespace strimzi & kafka are in, run the cmd with the followin
...
+
+and uninstall it with:
+
+```sh
+make uninstall/observatorium
+```
+
+#### Tenant tokens
+
+A default tenant with the name `test` is created. To obtain a token for this tenant:
```
Comment (david-martin) [inline README.md:119]: @pb82 What to do with the token?

**Comment by redmikhail** [comment]
Not sure what is protocol so I will just leave a comment , sorry :(. I have tested install/uninstall of observatorium , it works fine with exception that route for the observatorium API has not been created automatically. Maybe it is on purpose and it is probably fine for testing locally on the same cluster but at the same time there is no harm in creating it for posterity . In  README for DEX_ROUTE curl request environment variable I think should specify ${DEX_ROUTE} instead of $(DEX_ROUTE). We may want to expand README on how to setup grafana, loki and prometheus to send metrics/logs and grafana datasources (couldn't find in the document url either , had to improvise) . While makefile target `setup/observatorium` makes decent attempt to populate url , it doesn't create configmaps `observatorium-xyz-tls` in either prometheus or promtail and it may not be clear what to do with token. It probably should be captured in README or separate document

**Comment by pb82** [comment]
@redmikhail README updated with the ${DEX_ROUTE}. I'd prefer to add the other docs in a separate PR, same as the route.

**Review by redmikhail — APPROVED** [review]
LGTM

**Pull request merged**